sdk/identity/identity/test/manual-integration/AzureFunctions/IdentityTest/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "azure-function-test",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc",
+    "build:production": "npm run prestart && npm prune --production",
+    "watch": "tsc --w",
+    "prestart": "npm run build && func extensions install",
+    "start:host": "func start --typescript",
+    "start": "npm-run-all --parallel start:host watch",
+    "test": "echo \"No tests yet...\""
+  },
+  "description": "",
+  "devDependencies": {
+    "@azure/functions": "^1.0.1-beta1",
```

Comment:
There's not a stable azure/functions package?

Reply:
1.2.3 @azure/functions